terraso_backend/apps/soil_id/models/depth_dependent_soil_data.py

```diff
@@ -13,13 +13,273 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program. If not, see https://www.gnu.org/licenses/.
 
+from django.core.validators import MaxValueValidator, MinValueValidator
 from django.db import models
+from django.utils.translation import gettext_lazy as _
 
 from apps.core.models.commons import BaseModel
 from apps.soil_id.models.soil_data import SoilData
 
+OTHER = "other"
```

Comment:
nit: i'm not sure if there's any negative implications of sharing a value between different choice fields like this? @paulschreiber do you know?

Reply:
Using OTHER twice seems fine in this case.